src/pwncore/routes/team.py

```diff
@@ -9,9 +11,12 @@
 
 
 @router.get("/list")
-async def team_list():
-    # Do login verification here
-    return [{"team_name": "CID Squad"}, {"team_name": "Astra"}]
+async def team_list(response: Response):
+    teams = await Team.all().values()
+    if not teams:
```

Comment:
No need to check for teams being empty, if there are no teams return the empty list.